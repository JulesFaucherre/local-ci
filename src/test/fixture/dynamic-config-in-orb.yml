version: 2.1

setup: true
orbs:
  creating-config:
    orbs:
      continuation: circleci/continuation@0.1.2
    jobs:
      set-up-config:
        executor: continuation/default
        steps:
          - checkout
          - run:
              name: Create config
              command: |
                ./create-config > created_config.yml
          - continuation/continue:
              configuration_path: created_config.yml

jobs:
  example:
    executor:
      name: node
    steps:
      - node --version

workflows:
  setup:
    jobs:
      - creating-config/set-up-config
