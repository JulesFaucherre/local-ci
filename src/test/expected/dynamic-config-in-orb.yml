version: 2.1

setup: true
orbs:
  creating-config:
    orbs:
      continuation: circleci/continuation@0.1.2
    jobs:
      set-up-config:
        executor: continuation/default
        steps:
          - checkout
          - run:
              name: Create config
              command: |
                ./create-config > created_config.yml
          - run:
              name: Continue the pipeline
              command: if [ -f /tmp/local-ci/config.yml ]; then rm /tmp/local-ci/config.yml; fi; cp created_config.yml /tmp/local-ci/config.yml;

jobs:
  example:
    executor:
      name: node
    steps:
      - node --version

workflows:
  setup:
    jobs:
      - creating-config/set-up-config
